pkgs/development/python-modules/azure-identity/default.nix

```diff
@@ -0,0 +1,51 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy38
+, lib
+
+# pythonPackages
+, azure-common
+, azure-core
+, azure-nspkg
+, cryptography
+, mock
+, msal
+, msal-extensions
+, msrest
+, msrestazure
+}:
+
+buildPythonPackage rec {
+  pname = "azure-identity";
+  version = "1.1.0";
+  disabled = isPy38;
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "zip";
+    sha256 = "1xn4nwi4vly8n3mmphv0wbdg9k55gsgmk3fdwma8rm3m3c7593hc";
+  };
+
+  propagatedBuildInputs = [
+    azure-common
+    azure-core
+    azure-nspkg
+    cryptography
+    mock
+    msal
+    msal-extensions
+    msrest
+    msrestazure
+  ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled. Please do this with all packages.

Reply:
comments have been added above the `doCheck = false;`s in the latest forced-push

I also added tests to the packages in which doing so is possible

```
kamado:~/Documents/nixpkgs$ nix-review rev HEAD
$ git -c fetch.prune=false fetch --force https://github.com/NixOS/nixpkgs master:refs/nix-review/0
From https://github.com/NixOS/nixpkgs
   4f6f0491a4d..0b7b588de24  master     -> refs/nix-review/0
$ git worktree add /home/kamado/.cache/nix-review/rev-ff892ee1f063b4696a6a94983e0e986509e3cbf3/nixpkgs 0b7b588de246397b6c5e717e68e7015e7a9a0766
Preparing worktree (detached HEAD 0b7b588de24)
Updating files: 100% (20628/20628), done.
HEAD is now at 0b7b588de24 Merge pull request #76904 from ivan/ivan-unmaintain
$ nix-env -f /home/kamado/.cache/nix-review/rev-ff892ee1f063b4696a6a94983e0e986509e3cbf3/nixpkgs -qaP --xml --out-path --show-trace
$ git merge --no-commit ff892ee1f063b4696a6a94983e0e986509e3cbf3
Updating 0b7b588de24..ff892ee1f06
Fast-forward
 maintainers/maintainer-list.nix                                   | 10 +++++++++
 pkgs/development/python-modules/azure-identity/default.nix        | 53 ++++++++++++++++++++++++++++++++++++++++++++++
 pkgs/development/python-modules/azure-keyvault-nspkg/default.nix  | 35 ++++++++++++++++++++++++++++++
 pkgs/development/python-modules/bandit/default.nix                | 44 ++++++++++++++++++++++++++++++++++++++
 pkgs/development/python-modules/javaobj-py3/default.nix           | 26 +++++++++++++++++++++++
 pkgs/development/python-modules/msal-extensions/default.nix       | 35 ++++++++++++++++++++++++++++++
 pkgs/development/python-modules/msal/default.nix                  | 36 +++++++++++++++++++++++++++++++
 pkgs/development/python-modules/names/default.nix                 | 36 +++++++++++++++++++++++++++++++
 pkgs/development/python-modules/oyaml/default.nix                 | 41 +++++++++++++++++++++++++++++++++++
 pkgs/development/python-modules/pyjks/default.nix                 | 37 ++++++++++++++++++++++++++++++++
 pkgs/development/python-modules/pysmb/default.nix                 | 36 +++++++++++++++++++++++++++++++
 pkgs/development/python-modules/requirements-detector/default.nix | 38 +++++++++++++++++++++++++++++++++
 pkgs/development/python-modules/twofish/default.nix               | 33 +++++++++++++++++++++++++++++
 pkgs/development/python-modules/viewstate/default.nix             | 36 +++++++++++++++++++++++++++++++
 pkgs/top-level/python-packages.nix                                | 31 +++++++++++++++++++++++++--
 15 files changed, 525 insertions(+), 2 deletions(-)
 create mode 100644 pkgs/development/python-modules/azure-identity/default.nix
 create mode 100644 pkgs/development/python-modules/azure-keyvault-nspkg/default.nix
 create mode 100644 pkgs/development/python-modules/bandit/default.nix
 create mode 100644 pkgs/development/python-modules/javaobj-py3/default.nix
 create mode 100644 pkgs/development/python-modules/msal-extensions/default.nix
 create mode 100644 pkgs/development/python-modules/msal/default.nix
 create mode 100644 pkgs/development/python-modules/names/default.nix
 create mode 100644 pkgs/development/python-modules/oyaml/default.nix
 create mode 100644 pkgs/development/python-modules/pyjks/default.nix
 create mode 100644 pkgs/development/python-modules/pysmb/default.nix
 create mode 100644 pkgs/development/python-modules/requirements-detector/default.nix
 create mode 100644 pkgs/development/python-modules/twofish/default.nix
 create mode 100644 pkgs/development/python-modules/viewstate/default.nix
$ nix-env -f /home/kamado/.cache/nix-review/rev-ff892ee1f063b4696a6a94983e0e986509e3cbf3/nixpkgs -qaP --xml --out-path --show-trace --meta
$ nix build --no-link --keep-going --max-jobs 4 --option build-use-sandbox true -f /home/kamado/.cache/nix-review/rev-ff892ee1f063b4696a6a94983e0e986509e3cbf3/build.nix
[1 built, 0.0 MiB DL]
34 package were built:
python27Packages.azure-keyvault-nspkg python27Packages.javaobj-py3 python27Packages.msal python27Packages.msal-extensions python27Packages.names python27Packages.oyaml python27Packages.pyjks python27Packages.pysmb python27Packages.twofish python37Packages.azure-identity python37Packages.azure-keyvault-nspkg python37Packages.bandit python37Packages.javaobj-py3 python37Packages.msal python37Packages.msal-extensions python37Packages.names python37Packages.oyaml python37Packages.pyjks python37Packages.pysmb python37Packages.requirements-detector python37Packages.twofish python37Packages.viewstate python38Packages.azure-keyvault-nspkg python38Packages.bandit python38Packages.javaobj-py3 python38Packages.msal python38Packages.msal-extensions python38Packages.names python38Packages.oyaml python38Packages.pyjks python38Packages.pysmb python38Packages.requirements-detector python38Packages.twofish python38Packages.viewstate

$ nix-shell /home/kamado/.cache/nix-review/rev-ff892ee1f063b4696a6a94983e0e986509e3cbf3/shell.nix
innovation

[nix-shell:~/.cache/nix-review/rev-ff892ee1f063b4696a6a94983e0e986509e3cbf3]$ 
```